src/lib/components/collapsiblepanel/CollapsiblePanel.component.js

```diff
@@ -0,0 +1,114 @@
+//@flow
+import React from "react";
+import { lighten } from "polished";
+import styled, { css } from "styled-components";
+import Button from "../button/Button.component";
+import { mergeTheme } from "../../utils";
+import * as defaultTheme from "../../style/theme";
+
+type Props = {
```

Comment:
Please stick with what is defined in the ticket description:
`header` could be an array of Node (since the content could be icon, string, or anything) as you suggested :)
`
type Props = {
  header: Array<Node>, 
  onHeaderClick: any => void,
  expanded: boolean,
  children: Node, //content
};
`

Reply:
`variant` is a good idea. We can manage it later in another PR :)